BUILD.md

```diff
@@ -31,26 +31,32 @@ the latest [SBT] version following their websites' instructions or
 install them using your operating system's package manager.
 
 Since Daffodil now has a C backend as well as a Scala backend, you
-will need a C compiler supporting the [C99] standard or later, the
-[Mini-XML] library, and possibly the GNU [argp] library if your
-system's C library doesn't include it already.  You can install either
-[gcc] or [clang] using your operating system's package manager.  If
-you can't install the [Mini-XML] library using your operating system's
-package manager, you'll have to build it from source.  We'll tell you
-how to do that on Windows but the commands should work on other
-operating systems too.
-
-You can set your environment variables "CC" and "AR" to the correct
+will need a C compiler supporting the [C99] standard or later and the
+[Mini-XML] library.  You can install either [gcc] or [clang] using
+your operating system's package manager.  If you can't install the
+[Mini-XML] library using your operating system's package manager,
+you'll have to build it from source.  We'll tell you how to do that on
+Windows but the commands should work on other operating systems too.
+
+You can set your environment variables `CC` and `AR` to the correct
 commands (or set them to `true` to disable C compilation altogether)
 if you don't want `sbt compile` to call your C compiler with `cc` and
 `ar` as the default commands.
 
-## Fedora 33
+## Fedora 34
 
-You can use the `dnf` package manager to install most of the build
-requirements needed to build Daffodil:
+You can use the `dnf` package manager to install most of the tools
+needed to build Daffodil:
 
-    sudo dnf install gcc git java-11-openjdk-devel make mxml-devel pkgconf
+    sudo dnf install clang git java-11-openjdk-devel llvm make mxml-devel pkgconf
+
+If you want to use clang instead of gcc, you'll have to set your
```

Comment:
You set it up so it is clang by default now? So is this instruction correct still, or does it have these settings now, and you have to flip them to use gcc now?